qa/L0_trace/opentelemetry_unittest.py

```diff
@@ -199,6 +214,36 @@ def _check_events(self, span_name, events):
             )
             self.assertFalse(all(entry in events for entry in compute_events))
 
+    def _check_events_cancel(self, events):
```

Comment:
Maybe instead of a new `_check_events_cancel` we can somehow refactor `_check_events` and use it?